packages/.eslintrc.json

```diff
@@ -23,7 +23,18 @@
         "lodash/import-scope": ["error", "member"],
         "unused-imports/no-unused-imports-ts": "warn",
         "@typescript-eslint/await-thenable": "error",
+        "@typescript-eslint/consistent-indexed-object-style": ["off", "record"],
         "@typescript-eslint/no-for-in-array": "error",
-        "@typescript-eslint/consistent-indexed-object-style": ["off", "record"]
+        "@typescript-eslint/no-implicit-any-catch": "error",
+        "@typescript-eslint/no-implied-eval": "error",
+        "@typescript-eslint/no-invalid-this": "error",
+        "@typescript-eslint/no-invalid-void-type": "error",
```

Comment:
I have a concern about this rule. https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/no-invalid-void-type.md

It bans the following code:

```ts
interface Interface {
  lambda: () => void;
}
```

Which I don't think it is necessary to do so